entry_types/scrolled/package/spec/frontend/features/sectionAtmoAudio-spec.js

```diff
@@ -0,0 +1,55 @@
+import {AtmoAudio} from 'frontend/AtmoAudio';
+import {useFakeMedia, fakeMediaRenderQueries} from 'support/fakeMedia';
+import {media} from 'pageflow/frontend';
+
+import {renderEntry, usePageObjects} from 'support/pageObjects';
+import {fakeParentWindow} from 'support';
+
+describe('section AtmoAudio', () => {
+  usePageObjects();
+  useFakeMedia();
+
+  beforeEach(() => {
+    fakeParentWindow()
+  });
+
+  function getAudioFileSeed({
+    id = 1,
+    permaId = 100,
+    basename = 'audio'
+  } = {}) {
+    return {
+      fileUrlTemplates: {
+        audioFiles: {
+          mp3: ':id_partition/:basename.mp3',
+          m4a: ':id_partition/:basename.m4a',
+          ogg: ':id_partition/:basename.ogg'
+        }
+      },
+      audioFiles: [
+        {id, permaId, isReady: true, basename}
+      ]
+    };
+  }
+
+
+  it('is posted when section becomes active', () => {
+    const {getSectionByPermaId} = renderEntry({
+      seed: {
```

Comment:
@tf can you help me prepare seed data for section with atmo audio file ?

Reply:
`atmoAudioFileId` needs to go in a `configuration` property:

```
{permaId: 11, configuration: {atmoAudioFileId: 100}}
```